arches_lingo/src/arches_lingo/components/login/LoginForm.vue

```diff
@@ -65,7 +65,7 @@ const submit = async () => {
 form {
     display: flex;
     flex-direction: column;
-    gap: 1rem;
+    gap: var(--p-content-gap);
```

Comment:
We use a lot of 1rem gaps, so I added a custom token for it in arches.

Reply:
But as I mentioned [here](https://github.com/archesproject/arches/pull/11319#discussion_r1754464331) it's a little debatable and we can totally revert this.